src/docfx/lib/HtmlUtility.cs

```diff
@@ -298,11 +298,11 @@ public static bool IsInlineImage(this HtmlBlock node, int imageIndex)
             {
                 if (attribute.NameIs("data-raw-html"))
                 {
-                    rawHtml = HttpUtility.HtmlDecode(attribute.Value.ToString());
+                    rawHtml = attribute.Value.ToString();
```

Comment:
why no longer decoding here?

Reply:
because it'll be encoded anyway in line 329